packages/material-ui/src/Chip/Chip.test.js

```diff
@@ -85,11 +84,12 @@ describe('<Chip />', () => {
       assert.strictEqual(wrapper.props().tabIndex, 0);
     });
 
-    it('should apply user value of tabIndex', () => {
+    it('should apply user values of tabIndex and role', () => {
       wrapper = shallow(
         // eslint-disable-next-line jsx-a11y/tabindex-no-positive
-        <Chip onClick={() => {}} tabIndex={5} />,
+        <Chip onClick={() => {}} role="presentation" tabIndex={5} />,
       );
+      assert.strictEqual(wrapper.props().role, 'presentation');
```

Comment:
I think we shouldn't test prop forwarding here. We can get rid of setting `role="presentation"` and this assertion.